workspaces/adventure-pack/src/app/__tests__/__snapshots__/equip-test.ts.snap

```diff
@@ -2507,13 +2511,14 @@ function isNonNullish(value) {
 }
 
 function* traverseLevelOrder(root) {
-  if (!root) {
+  if (root == null) {
```

Comment:
Quick question why the preference from `!root` vs `root == null` Is this to be more specific since !variable checks for any `falsy` values?

Reply:
I've gone back and forth on this one. `== null` is more explicit, `!root` is more concise... Either one could be something to optimize for in library code. Today I'm feeling more explicit 😄